packages/desktop-client/src/components/sidebar/ItemContent.tsx

```diff
@@ -28,16 +25,14 @@ function ItemContent({
     <View
       role="button"
       tabIndex={0}
-      style={[
-        style,
-        {
-          touchAction: 'auto',
-          userSelect: 'none',
-          userDrag: 'none',
-          cursor: 'pointer',
-          ...(forceActive ? activeStyle : {}),
-        },
-      ]}
+      style={{
+        ...style,
+        touchAction: 'auto',
+        userSelect: 'none',
+        // userDrag: 'none',
```

Comment:
Intentional change?

Reply:
This doesn't seem to be a valid css property and compiler is complaining so I commented it out. But compiler is no longer complaining so I'll revert :)